web/source/kmwkeyboards.ts

```diff
@@ -1230,6 +1238,13 @@ class KeyboardManager {
       this.deferredKR.push(Pk);
       return;
     }
+
+    if(Pk['_kmw']) {
+      console.error("The keyboard _kmw property is a reserved field for engine use only; this keyboard is invalid.");
```

Comment:
Can we log the id of the keyboard as 'this keyboard' may be ambiguous when loading multiple keyboards...